bundles/org.openhab.binding.luftdateninfo/src/main/java/org/openhab/binding/luftdateninfo/internal/dto/SensorData.java

```diff
@@ -0,0 +1,83 @@
+/**
+ * Copyright (c) 2010-2020 Contributors to the openHAB project
+ *
+ * See the NOTICE file(s) distributed with this work for additional
+ * information.
+ *
+ * This program and the accompanying materials are made available under the
+ * terms of the Eclipse Public License 2.0 which is available at
+ * http://www.eclipse.org/legal/epl-2.0
+ *
+ * SPDX-License-Identifier: EPL-2.0
+ */
+package org.openhab.binding.luftdateninfo.internal.dto;
+
+import java.util.List;
+
+/**
+ * The {@link SensorData} Data Transfer Object
+ *
+ * @author Bernd Weymann - Initial contribution
+ */
+public class SensorData {
+    private Long id;
+    private String timestamp;
+    private Integer sampling_rate;
+    private List<SensorDataValue> sensordatavalues;
+    private Location location;
+    private Sensor sensor;
+
+    @Override
+    public String toString() {
+        return id + timestamp;
+    }
+
+    public Long getId() {
+        return id;
+    }
+
+    public void setId(Long id) {
+        this.id = id;
+    }
+
+    public String getTimeStamp() {
+        return timestamp;
+    }
+
+    public void setTimeStamp(String timeStamp) {
+        this.timestamp = timeStamp;
+    }
+
+    public Integer getSampling_rate() {
+        return sampling_rate;
+    }
+
+    public void setSampling_rate(Integer sampling_rate) {
+        this.sampling_rate = sampling_rate;
+    }
+
+    public List<SensorDataValue> getSensordatavalues() {
```

Comment:
See above

Reply:
Same here - field is exactly named after Json key

Reply:
Solved with SerializedName annotation